desertbot/modules/commands/RSS.py

```diff
@@ -0,0 +1,188 @@
+            return self.help(None)
+
+    def _getLatest(self, feedName):
+        lowerMap = {name.lower(): name for name in self.feeds}
+        if feedName.lower() in lowerMap:
+            name = lowerMap[feedName.lower()]
+            title = self.feeds[name]["latestTitle"]
+            link = self.feeds[name]["latestLink"]
+            return {
+                "name": name,
+                "title": title,
+                "link": link
+            }
+        else:
+            return None
+
+    def checkFeeds(self, message: IRCMessage):
+        responses = []
+        for feedName, feedDeets in self.feeds.items():
+            if feedDeets["lastCheck"] > datetime.datetime.utcnow() - datetime.timedelta(minutes=10):
+                continue
+
+            self.feeds[feedName]["lastCheck"] = datetime.datetime.utcnow()
+
+            response = self.bot.moduleHandler.runActionUntilValue("fetch-url", feedDeets["url"])
+
+            if not response:
+                self.logger.warning("failed to fetch {!r}, either a server hiccup "
+                                    "or the feed no longer exists".format(feedDeets["url"]))
+                continue
+
+            soup = BeautifulSoup(response.content, "lxml")
+            item = soup.find("item")
+
+            if item is None:
+                self.logger.warning("the feed at {!r} doesn't have any items, has it shut down?"
+                                    .format(feedDeets["url"]))
+                continue
+
+            itemDate = item.find("pubdate").text
+            newestDate = dparser.parse(itemDate, fuzzy=True, ignoretz=True)
+
+            if newestDate > feedDeets["lastUpdate"]:
+                self.feeds[feedName]["lastUpdate"] = newestDate
+                title = item.find("title").text
+                link = item.find("link").text
+                link = self.bot.moduleHandler.runActionUntilValue("shorten-url", link)
+                self.feeds[feedName]["lastTitle"] = title
+                self.feeds[feedName]["lastLink"] = link
+                if not feedDeets["suppress"]:
+                    response = "New {}! Title: {} | {}".format(feedName, title, link)
+                    responses.append(IRCResponse(ResponseType.Say, response, message.replyTo))
+                self.bot.storage["rss_feeds"] = self.feeds
+
+        return responses
+
+    @admin("[RSS] Only my admins may follow new RSS feeds!")
+    def _followFeed(self, message: IRCMessage):
+        try:
+            url = message.parameterList[1]
+            name = " ".join(message.parameterList[2:])
+            feed_object = {
+                "url": url,
+                "lastUpdate": datetime.datetime.utcnow() - datetime.timedelta(days=365),
+                "lastTitle": "",
+                "lastLink": "",
+                "lastCheck": datetime.datetime.utcnow() - datetime.timedelta(minutes=10),
+                "suppress": False
+            }
+            self.feeds.append({name: feed_object})
```

Comment:
dicts don't have an append method, I guess you just want self.feeds[name] = feed_object